packages/react-components/source/scss/library/components/_tabs.scss

```diff
@@ -5,6 +5,7 @@
   flex: 1;
   flex-direction: column;
   margin-top: $puppet-common-spacing-base * 4;
+  width: 100%;
```

Comment:
What if you did max-width: 100%? Would be more safe

Reply:
👍 Okay, it's `max-width: 100%` now.